libs/SalesforceSwiftSDK/SalesforceSwiftSDK/Extensions/SFRestAPIExtensions.swift

```diff
@@ -150,7 +150,7 @@ extension SFRestAPI {
          - Returns: SFRestRequest wrapped in a promise.
          */
         public func versions() -> Promise<SFRestRequest> {
```

Comment:
(I'm probably a bit late for that comment) 
But what's the value of having promises for synchronous APIs.
Why is 
```swift
restApi.Promises.resources()
          .then { (request) in
             restApi.send(request)
          }
```
Better than
```swift
restApi.send(restApi.requestForResources())
```

Reply:
@wmathurin Just a matter of  convenience, it allows you to start a promise chain from any of these factory methods. As for smartstore it also does offer a fluent api.